crates/sui-indexer/migrations/2023-10-06-204335_tx_indices/up.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE tx_senders (
+    cp_sequence_number          BIGINT       NOT NULL,
+    tx_sequence_number          BIGINT       NOT NULL,
+    -- SuiAddress in bytes.
+    sender                      BYTEA        NOT NULL,
+    PRIMARY KEY(sender, tx_sequence_number, cp_sequence_number)
+);
+CREATE INDEX tx_senders_tx_sequence_number_index ON tx_senders (tx_sequence_number ASC, cp_sequence_number ASC);
```

Comment:
when issuing a query, will we need to include `order by cp_sequence_number asc` or `desc` in order to leverage this index on scan or scan backward?

Reply:
this index can be used for both forward and backward scans.
https://www.postgresql.org/docs/current/indexes-ordering.html
> The index can also be scanned backward ...

in fact, it's a bit verbose here to specify `ASC` as it's the default one, I will remove it in a following PR I am working on